dll/win32/shell32/folders/CDrivesFolder.cpp

```diff
@@ -796,11 +795,9 @@ HRESULT WINAPI CDrivesFolder::CompareIDs(LPARAM lParam, PCUIDLIST_RELATIVE pidl1
             hres = MAKE_COMPARE_HRESULT(Diff.QuadPart);
             break;
         }
-        case 4:        /* comments */
+        case IDS_SHV_COLUMN_COMMENTS:
             hres = MAKE_COMPARE_HRESULT(0);
             break;
-        default:
-            return E_INVALIDARG;
     }
```

Comment:
Would it make sense to add
DEFAULT_UNREACHABLE;
macro here to calm down static analysis tools? Same question for the other 2 switch-cases that have been touched.

Reply:
I already added them before you have edited the comment (in fact I wanted to merge it already just before that), but if people will decide to not use `DEFAULT_UNREACHABLE;`, it's easy to remove.

Reply:
Okay, Thomas Faber had a look at it and also found, that DEFAULT_UNREACHABLE; is a good idea here.
That is enough authority for me. I do resolve this now and feel the PR to be ready for merge.